frontend/integration-tests/views/olm-catalog.view.ts

```diff
@@ -0,0 +1,27 @@
+import { browser, $, $$, by, element, ExpectedConditions as until } from 'protractor';
+
+import * as crudView from './crud.view';
+
+const rowSelector = '.co-resource-list__item';
+export const entryRows = $$(rowSelector);
+export const entryRowFor = (name: string) => element(by.cssContainingText(rowSelector, name));
+
+export const isLoaded = () => browser.wait(until.presenceOf($('.loading-box__loaded')), 10000).then(() => browser.sleep(500));
+
+export const clickTab = (tab: string) => crudView.clickTab(tab);
```

Comment:
I suggest just using the crud view implementation directly instead of creating an alias